projects/components/src/tabs/content/tab-group.component.scss

```diff
@@ -75,9 +75,25 @@
 .tab-label {
   flex: 1;
   display: flex;
-  flex-direction: column;
-  justify-content: center;
-  text-align: center;
+  flex-direction: row;
+  align-items: center;
+
+  .sub-text {
+    @include font-label($gray-5);
+    display: flex;
+    justify-content: center;
+    text-align: center;
+    margin-left: 6px;
+    width: 20px;
```

Comment:
fixed width might not scale well if the numbers are big right? I'd recommend against using fix width. Use padding to fill the area around.

Reply:
fixed